test_scripts/Defects/8_0/3674_2_HMI_starts_PTU_before_UpdateAppList.lua

```diff
@@ -153,6 +149,20 @@ local function registerAppWithPTU(pAppId, pMobConnId)
             { hmiLevel = "NONE", audioStreamingState = "NOT_AUDIBLE", systemContext = "MAIN" })
           session:ExpectNotification("OnPermissionsChange")
           :Times(AnyNumber())
+          local policyMode = SDL.buildOptions.extendedPolicy
+          if policyMode == policyModes.P or policyMode == policyModes.EP then
+            local lsEvent = common.run.createEvent()
+            common.getHMIConnection():ExpectEvent(lsEvent, "Lock Screen URL event")
+            session:ExpectNotification("OnSystemRequest")
+            :Do(function(_, d2)
+                if d2.payload.requestType == "LOCK_SCREEN_ICON_URL" then
```

Comment:
Maybe we need to add check for requestType in `policyTableUpdateHttp` function
`common.getMobileSession():ExpectNotification("OnSystemRequest")`

Reply:
I believe that is a redundant check for the mentioned issue